index.ts

```diff
@@ -21,21 +27,35 @@ export const setupPlugin: AvoInspectorPlugin['setupPlugin'] = async ({ config, g
         'content-type': 'application/json',
         accept: 'application/json',
     }
+
+    global.excludeEvents = new Set(
+        config.excludeEvents ? config.excludeEvents.split(',').map((event) => event.trim()) : null
+    )
+    global.includeEvents = new Set(
+        config.includeEvents ? config.includeEvents.split(',').map((event) => event.trim()) : null
+    )
+    global.excludeProperties = new Set(
+        config.excludeProperties ? config.excludeProperties.split(',').map((event) => event.trim()) : null
+    )
+    global.includeProperties = new Set(
+        config.includeProperties ? config.includeProperties.split(',').map((event) => event.trim()) : null
+    )
 }
 
-export const onEvent: AvoInspectorPlugin['onEvent'] = async (event, { config, global }) => {
-    if (event.event.startsWith("$")) {
+export const composeWebhook: AvoInspectorPlugin['onEvent'] = async (event, { config, global }) => {
+    const isIncluded = global.includeEvents.length > 0 ? global.includeEvents.includes(event.event) : true
+    const isExcluded = global.excludeEvents.includes(event.event)
+
+    if (event.event.startsWith("$") && isIncluded && !isExcluded) {
```

Comment:
We want the users to be able to filter in/out any events not just PostHog events
```suggestion
    if (isIncluded && !isExcluded) {
```

Reply:
The current implementation skips PostHog events, doing this would not be backwards compatible. I rather fix one problem than potentially cause another.

Reply:
🤔  currently we ignore events if they are `$` and isIncluded is true and excluded is false ... that doesn't seem right...

We'll likely want:
```
    // by default exclude PostHog events
    const isIncluded = global.includeEvents.length > 0 ? global.includeEvents.includes(event.event) : !event.event.startsWith("$")
    const isExcluded = global.excludeEvents.includes(event.event)
    // if the same event is in both lists, we default to not sending it for efficiency reasons
    if (!isIncluded || isExcluded) { 
      return
    }
```

Reply:
Oh yeah, there should be an OR there. Still against changing existing functionality, but that's a bug.

Reply:
it doesn't change the existing implementation if you make `!event.event.startsWith("$")` the default as I did above (if one doesn't set isIncluded which is the case before your PR then they'd get existing behaviour ;)

Reply:
Just pushed a commit to fix the bug, and that's what happens now.

Reply:
I really worry that we'll need to come back here as some folks will want to send Pageview events for example. We can seamlessly update it now, because it wouldn't change anything for anyone, but we can't do that once we have some users for `$isIncluded` already as I mentioned above

Reply:
> I really worry that we'll need to come back here as some folks will want to send e.g. $pageview events for example.


That's not currently possible, so we would have had to come back anyways.

Reply:
If you wish to make a breaking change, go ahead and take over my PR.

Reply:
Sure, ship as it (note properties had the same problem)
But I'm confused, so before I put up a follow-up PR maybe you can help me understand how that would be a breaking change. Here's my understanding of the world:

Currently: we don't send events that start with `$`, all other events are sent

this change: we don't send events that start with `$`, for others if a user sets `isIncluded`, then we send those events, except if a user adds an event that starts with `$` in `isIncluded`, then we still won't send it.

my proposal: we don't send events that start with `$` by default (i.e. all current users get this behaviour), if a user sets `isIncluded` they choose explicitly which events to send, this can include events that start with `$`.